src/renderer/components/TaskItem/index.tsx

```diff
@@ -45,9 +45,7 @@ export default function TaskItem({
       />
       <div className="col">
         <p className={styles['task-label']}>{label}</p>
-        <p className={styles['task-date']}>
-          {format(new Date(date), "E., dd 'de' MMM")}
-        </p>
+        <p className={styles['task-date']}>{new Date(date).toJSON()}</p>
```

Comment:
We can keep the format here.

Reply:
this issue not reproducible anymore so I think we should not merge this code as of now, wdy think?

Reply:
sure, you can revert this part, only the fix on `onSave()` solves the problem, we can keep the `format` function here.